### PR TITLE
Canonical press centre

### PR DESCRIPTION
**Pull request by carkod**
## Done

Canonical press centre and news section only.

Canonical press is a collection of news, resources, events etc... [See miro board](https://miro.com/app/board/o9J_luilNcw=/). Currently we only have data for news section.

## QA

- Check https://canonical-com-502.demos.haus/press-centre
- ~~check https://canonical-com-502.demos.haus/press-centre/news~~ (Reverted back to old ubuntu.com/blog/press-centre design, so no news page)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4874

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@carkod I think for now we could use the card style from the [Ubuntu Press Centre](https://ubuntu.com/blog/press-centre) including the category information.  And why not keep the cards on the right hand side? Unless of course @davegoddard42 thinks we shouldn't.

Could you also reduce the padding in the top strip, you can use the Ubuntu page as a reference.

Thanks.

**Comment by carkod** [comment]
> @carkod I think for now we could use the card style from the [Ubuntu Press Centre](https://ubuntu.com/blog/press-centre) including the category information. And why not keep the cards on the right hand side? Unless of course @davegoddard42 thinks we shouldn't.
> 
> Could you also reduce the padding in the top strip, you can use the Ubuntu page as a reference.
> 
> Thanks.

Yes, @davegoddard42 removed it from the wireframe [See miro board](https://miro.com/app/board/o9J_luilNcw=/).

**Comment by davegoddard42** [comment]
@carkod @spencerbygraves I think this might be a question to raise in the canonical.com sync meeting. I designed the wireframe based on research and benchmarking. I then had further input from Aldo based on his experience of working with agencies. 

I believe what's in the wireframe is the correct design, and would like it to be built. However, I realise that as a phased project there might be stepping stones. And replicating the Ubuntu Press Centre might be an easy first step.

Let's raise this in the sync meeting.

**Comment by spencerbygraves** [comment]
@carkod thanks for the updates, I've spotted a couple of things to change please:

- There is an overlap at the bottom of both the press centre and news pages:

image

image

- On news pages, could you reduce the padding in the top title area please. You can use the Ubuntu equivalent as a guide
- Again on news pages, are you able to make the images and sign up card align horizontally?

image

**Comment by carkod** [comment]
> @carkod thanks for the updates, I've spotted a couple of things to change please:
> 
> * There is an overlap at the bottom of both the press centre and news pages:
> 
> image
> 
> image
> 
> * On news pages, could you reduce the padding in the top title area please. You can use the Ubuntu equivalent as a guide
> * Again on news pages, are you able to make the images and sign up card align horizontally?
> 
> image



> @carkod thanks for the updates, I've spotted a couple of things to change please:
> 
> * There is an overlap at the bottom of both the press centre and news pages:
> 
> image
> 
> image
> 
> * On news pages, could you reduce the padding in the top title area please. You can use the Ubuntu equivalent as a guide
> * Again on news pages, are you able to make the images and sign up card align horizontally?
> 
> image

Fixed. Sorry, the news section has been removed, because the current https://ubuntu.com/blog/press-centre doesn't have such a section and it doesn't link to it

**Comment by spencerbygraves** [comment]
>Fixed. Sorry, the news section has been removed, because the current https://ubuntu.com/blog/press-centre doesn't have such a section and it doesn't link to it

Sorry, I wasn't very clear. By news section I meant the individual pages such as https://canonical-com-502.demos.haus/blog/inclusive-language-and-its-future-at-canonical

Thanks.

**Comment by carkod** [comment]
> > Fixed. Sorry, the news section has been removed, because the current https://ubuntu.com/blog/press-centre doesn't have such a section and it doesn't link to it
> 
> Sorry, I wasn't very clear. By news section I meant the individual pages such as https://canonical-com-502.demos.haus/blog/inclusive-language-and-its-future-at-canonical
> 
> Thanks.

Done.

**Comment by spencerbygraves** [comment]
@carkod thanks for updating, I'm still seeing this overlap:

image

**Comment by carkod** [comment]
> ## Done
> Canonical press centre and news section only.
> 
> Canonical press is a collection of news, resources, events etc... [See miro board](https://miro.com/app/board/o9J_luilNcw=/). Currently we only have data for news section.
> 
> ## QA
> * Check https://canonical-com-502.demos.haus/press-centre
> * ~check https://canonical-com-502.demos.haus/press-centre/news~ (Reverted back to old ubuntu.com/blog/press-centre design, so no news page)
> 
> ## Issue / Card
> Fixes [canonical-web-and-design/web-squad#4874](https://github.com/canonical-web-and-design/web-squad/issues/4874)

Sorry, take a look now please, I believe the demo was not working properly

**Comment by spencerbygraves** [comment]
@carkod I can still see the overlap, has the demo updated?

We may need to make that strip white with a border above it if the grey doesn't follow the angle of the footer.

Thanks.

File: templates/press-centre/index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      </div>
+
+      <div class="col-4">
+        <div class="p-card">
+          <h4>Logos</h4>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked" style="background-image:url('data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 width=%2214%22 height=%2214%22 viewBox=%220 0 14 14%22%3E%3Ccircle fill=%22%23333%22 cx=%227%22 cy=%227%22 r=%227%22/%3E%3Cpath fill=%22%23fff%22 d=%22M6.1 10.813L2.41 8.105l1.184-1.613L5.9 8.187l4.393-4.394 1.414 1.414z%22 /%3E%3C/svg%3E');"><a href="https://assets.ubuntu.com/v1/18a64171-ubuntu-brandmark1.zip" title="Ubuntu logo pack zip file">Ubuntu logo pack [ZIP]&nbsp;&rsaquo;</a></li>
+            <li class="p-list__item is-ticked" style="background-image:url('data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 width=%2214%22 height=%2214%22 viewBox=%220 0 14 14%22%3E%3Ccircle fill=%22%23333%22 cx=%227%22 cy=%227%22 r=%227%22/%3E%3Cpath fill=%22%23fff%22 d=%22M6.1 10.813L2.41 8.105l1.184-1.613L5.9 8.187l4.393-4.394 1.414 1.414z%22 /%3E%3C/svg%3E');"><a href="https://assets.ubuntu.com/v1/47b9260c-canonical-brandmark1.zip" title="Canonical logo pack">Canonical logo pack [ZIP]&nbsp;&rsaquo;</a></li>
```
Comment (sowasred2012) [inline templates/press-centre/index.html:32]: Should we update these two zip files with the new brand assets? A question for @waynecrosby I think.

File: templates/blog/newsletter-form.html [hunk 2]
```diff
@@ -1,96 +1,86 @@
 <div class="p-card--muted">
-    <header class="p-card__header">
-      <h5 class="p-muted-heading">Newsletter signup</h5>
-    </header>
+  <header class="p-card__header">
+    <h5 class="p-muted-heading">Newsletter signup</h5>
```
Comment (sowasred2012) [inline templates/blog/newsletter-form.html:3]: Again, I think this is just how it is on the u.c press centre & blog, but the heading heirarchy is all over the map on the blog post view:
Screenshot from 2022-04-01 11-01-16

It should be:
- 1: Title of blog post
  - 2: Newsletter signup
    - 3: Select topics you're interested in 
  - 2: Related blog posts
    - 3: Title of related post
    - 3: Title of related post
    - 3: Title of related post

And I don't think the author "Canonical" should be a heading element at all, that could be a `p` element, with a `.p-heading--3` if necessary.

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Mostly LGTM - there's a lot of comments, but they're all about heading heirarchy, so it looks more dramatic than it is.

File: templates/press-centre/card.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    </p>
+  </header>
+  {% endif %}
+  
+  <div class="p-card__content">
+    <h2 class="p-heading--4">
+      <a href="/blog/{{ article.slug }}">{{ article.title.rendered|safe }}</a>
+    </h2>
+    <p>
+      <em>by<a href="/blog/author/{{article.author.slug}}">{{ article.author.name }}</a>on {{ article.date }}</em>
```
Comment (sowasred2012) [inline templates/press-centre/card.html:20]:
```suggestion
      <em>by <a href="/blog/author/{{article.author.slug}}">{{ article.author.name }}</a> on {{ article.date }}</em>
```

File: templates/press-centre/index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+              <li class="p-inline-list__item"><a href="/contact-us">Contact Canonical</a></li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="p-card">
+          <h2 class="p-heading--4">Archives</h2>
+          <ul class="p-list">
+            {% for year in descending_years(2015) %}
+              <li class="p-list__item"><h5><a class="p-link--soft" href="/blog/archives?group=canonical-announcements&amp;year={{ year }}">{{ year }}</a></h5></li>
```
Comment (sowasred2012) [inline templates/press-centre/index.html:99]:
```suggestion
              <li class="p-list__item"><h3 class="p-heading--5"><a class="p-link--soft" href="/blog/archives?group=canonical-announcements&amp;year={{ year }}">{{ year }}</a></h3></li>
```

**Review by sowasred2012 — APPROVED** [review]
LGTM - two more suggestions, but good to merge when they're addressed.

File: templates/press-centre/index.html (hunk 1 above)
Comment (carkod) [reply templates/press-centre/index.html:32]: He needs to take a look before we release new assets

**Pull request merged**